src/main/scala/mesosphere/marathon/MarathonModule.scala

```diff
@@ -75,6 +77,14 @@ class MarathonModule(conf: MarathonConf, http: HttpConf, zk: ZooKeeperClient)
 
   }
 
+  @Named(ModuleNames.NAMED_HTTP_EVENT_STREAM)
+  @Provides
+  @Singleton
+  def provideHttpEventStreamActor(system: ActorSystem,
+                                  @Named(EventModule.busName) eventBus: EventStream): ActorRef = {
+    system.actorOf(Props(classOf[HttpEventStreamActor], eventBus))
```

Comment:
Ok. There is no unique identifier. I need to create a synthetic one. Will add the remote address as one part of that.